app/portainer/settings/authentication/internal-auth/PasswordLengthSlider.tsx

```diff
@@ -0,0 +1,100 @@
+import { useEffect, useState } from 'react';
+import RcSlider from 'rc-slider';
+import clsx from 'clsx';
+
+import { Badge } from '@/portainer/components/Badge/Badge';
+
+import 'rc-slider/assets/index.css';
+
+import styles from './PasswordLengthSlider.module.css';
+
+export interface Props {
+  min: number;
+  max: number;
+  step: number;
+  defaultValue: number;
+}
```

Comment:
every form element should have `value` and `onChange` Props, to pass the values to the parent form.